hnn_core/gui/gui.py

```diff
@@ -1319,6 +1390,17 @@ def _init_network_from_widgets(params, dt, tstop, single_simulation_data,
             synch_inputs_kwargs['n_drive_cells'] = 1
             synch_inputs_kwargs['cell_specific'] = False
 
+        if drive['type'] in ('Tonic'):
+            weights_amplitudes = {
+                k: v.value
+                for k, v in drive["amplitude"].items()
+            }
+            single_simulation_data['net'].add_tonic_bias(
+                amplitude=weights_amplitudes,
+                t0=drive["t0"].value,
+                tstop=drive["tstop"].value)
+            return
```

Comment:
Should this be returning?

Reply:
The rest of the logic in that function doesnt apply to Tonic inputs. I want to avoid the `elif` branching

Reply:
We talked it over and I'll remove the return